content/programming_model_overview.tex

```diff
@@ -157,7 +157,7 @@
       of concatenated symmetric objects contributed by each of the \acp{PE} in
       another symmetric data object.
   \item \OPR{Reduction}: All \acp{PE} participating in the routine get the result
-      of an associative binary routine over elements of the specified symmetric
+      of a binary operation over elements of the specified symmetric
```

Comment:
Is it required that the destination buffers on all PEs be bit for bit identical?

Reply:
@davidozog We need to keep "associative" or we'll be losing important semantics. I suggest we leave this sentence unmodified and just live with the inconsistency for OpenSHMEM 1.6.

Reply:
@lstewart Agree -- that's not specified and something that we should clarify in OpenSHMEM 1.7.

Reply:
@jdinan - sure, I'm good with that approach.  This change is now reverted.